src/box/box.cc

```diff
@@ -3491,13 +3497,19 @@ bootstrap_from_master(struct replica *master)
 	 */
 
 	assert(!tt_uuid_is_nil(&INSTANCE_UUID));
-	enum applier_state wait_state = APPLIER_WAIT_SNAPSHOT;
-	applier_resume_to_state(applier, wait_state, TIMEOUT_INFINITY);
+	try {
+		applier_resume_to_state(applier, APPLIER_FETCH_SNAPSHOT,
```

Comment:
Should be `APPLIER_CONTINUE_SNAPSHOT` as we discussed?

Reply:
> Should be `APPLIER_CONTINUE_SNAPSHOT` as we discussed?
We wanted to make a change ([comment](https://github.com/tarantool/tarantool/pull/7183#discussion_r907215400)), didn't we?

Reply:
Yes, indeed, I got confused. Sorry.